### PR TITLE
Updates to Leapsight entry

### PR DESCRIPTION
**Pull request by aramallo**
TODO: 
- [X] I have read the [contribution guide](CONTRIBUTING.md). 
- [X] I have checked my spelling and grammar
- [X] I :heart: Erlang/OTP

### CODE REVIEW

File: src/_data/companies.yml [hunk 1]
```diff
@@ -269,12 +269,12 @@ LambdaClass:
...
 Leapsight:
-  industry: Information Technology
+  industry: Information Technology, IoT
   url: http://www.leapsight.com
   blog: http://www.leapsight.com/
   github: https://gitlab.com/leapsight
-  location: London, United Kingdom; Buenos Aires, Argentina; Rosario, Argentina
-  description: We use Erlang to do scalable, interactive, intelligent systems. Erlang also powers our products - Bondy, an OSS WAMP Broker and API Gateway; and LSD, a distributed, semantic and deductive database.
+  location: London, United Kingdom; Buenos Aires, Argentina
+  description: We use Erlang to do scalable, real-time, intelligent systems. Erlang powers our products: Bondy (open source networking platform for MicroServices and IoT, a WAMP Router and REST API GAteway), LSD (distributed, knowledge graph and deductive database) and Leapsight IoT Platform.
```
Comment (starbelly) [inline src/_data/companies.yml]: Hi @aramallo . It seems as though the `:` in the sentence is making jekyll's yaml parser unhappy. 

```
jekyll 3.8.6 | Error:  (/home/travis/build/starbelly/erlang-companies/src/_data/companies.yml): mapping values are not allowed in this context at line 277 column 104
```

Comment (aramallo) [reply src/_data/companies.yml]: Thanks @starbelly , I will fix that

**Comment by starbelly** [comment]
Thank you for the update!! 🐸

**Comment by aramallo** [comment]
Thanks a lot!

**Pull request merged**